cs/shipdesign.go

```diff
@@ -817,7 +823,10 @@ func DesignShip(techStore *TechStore, hull *TechHull, name string, player *Playe
 					break
 				}
 			}
-			// spare orbital slots left; use electrical items instead
+			// spare orbital slots left; use electrical items instead if possible
+			if hullSlot.Type&HullSlotTypeElectrical == 0 {
+				break
```

Comment:
Break will break out of the switch and ignore the component. I think you want to remove the if and let it fall through?

Reply:
No - this prevents it from putting computers in orbital only slots (since those fallthrough into the case for orbital/electrical)

Reply:
oh I see. I think I need to rearrange the order of those slots. That fall through shouldn't be possible and shouldn't require a break to fix, heh. Thanks!